docs/resources.md

```diff
@@ -115,13 +114,13 @@ rather than decimal ones: "64MiB" rather than "64MB".
 A resource type may have an associated read-only ResourceType structure, that contains metadata about the type.  For example:
 ```
 resourceTypes: [
-  "kubernetes.io/resources/memory": [
+  "resources.kubernetes.io/memory": [
     isCompressible: false, ... 
   ]
-  "kubernetes.io/resources/cpu": [
+  "resources.kubernetes.io/cpu": [
     isCompressible: true, internalScaleExponent: 3, ...
   ]
-  "kubernetes.io/resources/diskSpace": [ ... }
+  "resources.kubernetes.io/diskSpace": [ ... }
```

Comment:
Nothing else in Kubernetes uses this style of `bumpyCaps`. How about all lower case with hyphens, since it's a DNS label?